js/src/components/paid-ads/validateCampaign.js

```diff
@@ -1,26 +1,56 @@
 /**
  * External dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { __, sprintf } from '@wordpress/i18n';
 
 /**
  * @typedef {import('.~/components/types.js').CampaignFormValues} CampaignFormValues
  */
 
+// Minimum percentage of the recommended daily budget.
+const BUDGET_MIN_PERCENT = 0.3;
+
 /**
  * Validate campaign form. Accepts the form values object and returns errors object.
  *
  * @param {CampaignFormValues} values Campaign form values.
+ * @param {Object} opts Extra form options.
  * @return {Object} errors.
  */
-const validateCampaign = ( values ) => {
+const validateCampaign = ( values, opts ) => {
 	const errors = {};
 
 	if ( ! Number.isFinite( values.amount ) || values.amount <= 0 ) {
-		errors.amount = __(
-			'Please make sure daily average cost is greater than 0.',
-			'google-listings-and-ads'
-		);
+		return {
+			amount: __(
+				'Please make sure daily average cost is greater than 0.',
+				'google-listings-and-ads'
+			),
+		};
+	}
```

Comment:
@eason9487 Yes. It'll be useful to keep the validation in case there are no budget recommendations for the Ads account.
![image](https://github.com/user-attachments/assets/5a87695b-b4d8-44ce-8b01-81b8c7666589)

In that case, we simply check if the minimum amount is greater than 0.

Reply:
The confusion I meant was not about the case of no budget recommendations but was a scenario like:

1. 🧑‍💼: "Let's see what happens if I enter 0 for the daily budget"
2. UI shows `Please make sure daily average cost is greater than 0.`
3. 🧑‍💼: "Sure, that makes sense. I would like to set 3 for the daily budget"
4. UI shows `Please make sure daily average cost is greater than $5.00.`
5. 🧑‍💼: "How can there be two minimum values? Why didn't it tell me from the beginning that the value had to be at least 5?"

https://github.com/user-attachments/assets/9fc31baf-7cc7-4ecb-961a-a057f4d03843

Reply:
I just swapped the order of the checks. Let me know if this is better now.